mobile/src/main/java/ch/epfl/sdp/mobile/ui/profile/ProfileScreenState.kt

```diff
@@ -24,4 +24,7 @@ interface ProfileScreenState<C : ChessMatch> : Person {
    * @param match the [ChessMatch] to open.
    */
   fun onMatchClick(match: C)
+
+  /** Call back function to get back to previous screen */
+  fun onBack()
```

Comment:
This method should probably be defined on `VisitedProfileScreenState` only, since the settings screen does not have a back button.

Reply:
👍